### PR TITLE
Delete traffic-manager.md

### PR DESCRIPTION
**Pull request by msyyc**
Fix https://github.com/Azure/azure-sdk-for-python/issues/27830
The file has invalid button. `Sima Zhu` advise to delete it and pipeline will generate a new one:
https://github.com/Azure/azure-sdk-for-python/issues/27830#issuecomment-1378050307

### CODE REVIEW

**Review by sima-zhu — APPROVED** [review]

**Comment by sima-zhu** [comment]
@msyyc Could you resolve the conflicts? I can help you merge if you can't merge it.

**Comment by msyyc** [comment]
@sima-zhu I resolve the conflict. But CI will update the doc periodically and I don't have merge access. So I am afraid it will have conflict again if it can't be merged asap.

**Pull request merged**
